bootloader/prebootloader/src/tron/mbr/rtl_it.c

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
In general, do we want to rename the directory `tron` to `p2` ?

Reply:
Let's keep this as-is for now, just to avoid introducing any breakages at this point, we can always rename things later and for all intents and purposes everywhere else we use the P2 notation, just some folder names are leftovers.